index.html

```diff
@@ -1,362 +1,657 @@
+                        typesetting industry. Lorem Ipsum has been the
+                        industry's standard dummy text ever since the 1500s,
+                        when an unknown printer took a galley of type and
+                        scrambled it 1960s with the releaLorem Ipsum is simply
+                        dummy text of the printing and typesetting industry.
+                        Lorem Ipsum has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a
+                        galley of type and scrambled it 1960s with the releorem
+                        Ipsum is simply dummy text of the printing and
+                        typesetting industry. Lorem Ipsum han printer took a
+                        galley of type and scrambled it 1960s with the releawn
+                        printer took a galley of type and scrambled it 1960s
+                        with the releaLorem Ipsum is simply dummy text of the
+                        printing and typesetting industry. Lorem Ipsum has been
+                        the industry's standard dummy text ever since the 1500s,
+                        when an unknown printer took a galley of type and
+                        scrambled it 1960s with the relea
+                      </p>
+                      <p class="paragraph-description-mobile">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy
+                         text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+                        It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+
+                      </p>
+                    </div>
+                    <div class="action-content">
+                      <div class="popup-tags">
+                        <ul class="first">
+                          <li>HTML</li>
+                          <li>CSS</li>
+                          <li>javascript</li>
+                        </ul>
+                        <ul class="second">
+                          <li>github</li>
+                          <li>ruby</li>
+                          <li>Bootstrap</li>
+                        </ul>
+                      </div>
+                      <div class="divider-popup">
+                        <img src="images/Devider.svg" alt="Divider popup" />
+                      </div>
+                      <div class="popup-button-action">
+                        <button type="submit">
+                          <a href=""
+                            >See live
+                            <img
+                              src="images/deploy.svg"
+                              alt="deployment icon"
+                              class="deploy-icon-popup"
```

Comment:
- [ OPTIONAL ] From the project requirement, you should display all these elements dynamically using javascript :relaxed: so you need to remove all these from your HTML code. I think all this project information should be stored in an array of objects and iterated over displaying them dynamically.